### DISCUSSION TITLE
Headers are not included in the request execution at the Automatic docs "/docs"

### DISCUSSION THREAD

**Discussion by moemen95** [fastapi/fastapi#11076] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
@staticmethod
    def create(payload: EngineAPIPayload, authorization: Annotated[str | None, Header(alias="Authorization")]):
        pass
```

### Description

![Uploading image.png…]()

### Operating System

macOS

### Operating System Details

MacOS Ventura 13.6.2

### FastAPI Version

0.109.0

### Pydantic Version

2.5.3

### Python Version

3.11.5

**Comment by Kludex** [comment, marked as answer]
Duplicated.

- https://github.com/tiangolo/fastapi/discussions/11076
- https://github.com/tiangolo/fastapi/discussions/8613
- https://github.com/tiangolo/fastapi/discussions/7816
- https://github.com/tiangolo/fastapi/issues/612

**Reply by moemen95** [reply]
Thanks for pointing out.
